client/src/pages/Search.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import avatar from "../assets/dashboard/avatar.svg";
+import { FaSearch } from "react-icons/fa";
+import SearchProfileCard from "../components/SearchProfileCard";
+
+function Search() {
+  return (
+    <main className="lg:text-xl lg:text-lg">
```

Comment:
I think we could delete one of these rules

Reply:
😲 ok